src/components/page.tsx

```diff
@@ -31,12 +33,19 @@ type PageWrapperProps = {
 export const PageTop: React.FC<TitleProps> = ({ tx, ...props }) => {
   const status = tx.tx_status;
   const failed = status === 'abort_by_response' || status === 'abort_by_post_condition';
+  const networkMode = useNetworkMode();
+  // for testnet, show after 4 hours. for mainnet, show after 24 hours
+  const longPending =
+    dayjs().diff(dayjs.unix(tx.receipt_time), 'h') > (networkMode === 'testnet' ? 4 : 24);
 
   const failedMessage =
     status === 'abort_by_response'
       ? 'This transaction did not succeed because the transaction was aborted during its execution.'
       : 'This transaction would have succeeded, but was rolled back by a supplied post-condition.';
 
+  const longPendingMessage =
+    'Transactions that cannot be confirmed within 256 blocks will eventually be garbage-collected.';
```

Comment:
```suggestion
    'Transactions that cannot be confirmed within 256 blocks are eventually garbage-collected.';
```

Reply:
Is garbage collection a term we're using for stale unconfirmed txs?

Reply:
good point. maybe there's a better way to communicate that. I'm assuming some end-users might look at this and not be familiar with garbage collection. We could rather say "... are eventually removed from the mempool."

cc @markmhx or @aulneau ?

Reply:
Combining the feedback Kyran had earlier, here's what I changed it to:

```
'Transactions that cannot be confirmed within 256 blocks are eventually canceled automatically.'
```

(adding "eventually" because it is not exactly known when the collection will happen)